src/Combinatorics/Matroids/matroids.jl

```diff
@@ -1019,3 +1019,28 @@ function automorphism_group(M::Matroid)
   resize!(I, nrows(I), length(M))
   return automorphism_group(I; action=:on_cols)
 end
+
+@doc raw"""
+    is_quotient(Q1::Matroid,Q2::Matroid)
+
+Let Q1 and Q2 be matroids on the same groundset with rank $rank(Q1) \leq rank(Q2)$.  
+Check if $Q1$ is a matroid quotient of $Q2$.
+
+# Examples
+
+```jldoctest
+julia> Q1 = uniform_matroid(1,3)
+Matroid of rank 1 on 3 elements
+
+julia> Q2 = uniform_matroid(2,3)
+Matroid of rank 2 on 3 elements
+
+julia> is_quotient(Q1,Q2)
+true
+```
+"""
+function is_quotient(Q1::Matroid,Q2::Matroid)
```

Comment:
Another super nitpick, but: why are these matroids called Q1 and Q2, and not, say M1 and M2 ? I could understand if they were named `Q` and `M` becayse the first one is tested for being a *Q*uotient and hence called *Q*.

But again: not a blocker, feel free to ignore, I am just wondering as a bystander ;-)

Reply:
Fair point. I am in the habit of use Q for matroids, but not attached to it.